docs/source/packages/accounts-2fa.md

```diff
@@ -0,0 +1,140 @@
+---
+title: accounts-2fa
+description: Documentation of Meteor's `accounts-2fa` package.
+---
+
+The package allows you to easily integrate 2FA with the OTP technology on your login flow. It uses [node-2fa](https://www.npmjs.com/package/node-2fa) which works on top of [notp](https://github.com/guyht/notp), which implements TOTP ([RFC 6238](https://www.ietf.org/rfc/rfc6238.txt)) (the Authenticator standard), which is based on HOTP ([RFC 4226](https://www.ietf.org/rfc/rfc4226.txt)) to provide codes that are exactly compatible with all other Authenticator apps and services that use them.
+
+> This package is meant to be used with `accounts-password`, so if you still didn't add `account-password` to your project, you'll need to add it too. In the future we want to enable the use of this app with other login methods, like `accounts-passwordless` or our oauth methods (Google, GitHub, etc...).
+
+<h3 id="activating-2fa">Activating 2FA</h3>
+
+The first step to use 2FA is to generate a QR code so the user can read it on an authenticator app and start to receiving codes.
+
+{% apibox "Accounts.generate2faActivationQrCode" "module":"accounts-base" %}
+
+Receives an `appName` which is the name of your app that will show up when the user scans the QR code. Also, a callback called with a QR code in SVG format on success or a single `Error` argument
+on failure. Both parameters are optional.
```

Comment:
how they could be optional? what is going to happen if I can do this without any param? Is this case useful?

Reply:
This description is wrong. Just appName is optional. Fixed :palms_up_together: :